app/src/main/res/layout/fragment_edit_filter.xml

```diff
@@ -148,8 +149,7 @@
                     android:layout_height="wrap_content"
                     android:hint="@string/package_name"
                     app:boxCornerRadiusTopStart="10dp"
-                    app:boxCornerRadiusTopEnd="10dp"
-                    android:layout_marginEnd="20dp"
```

Comment:
I still believe at least small margin is needed. Otherwise ripple is too close to the text field